### DISCUSSION TITLE
RECUPERAR SENHA USER "SA" SQL SERVER EM DOCKER

### DISCUSSION THREAD

**Discussion by DrenokBR** [balta-io/forum#494] [Tira Dúvidas] [answered]
Fala Balta !
Boa noite !

Estou usando o SQL Server em Docker para os estudos, conforme sua dica, mas como eu estava estudando outras coisas, fiquei sem mexer com ele (SQL Server) uns dias e acabei esquecendo a senha do usuário "sa". Como essa não é uma instalação padrão do SQL Server, não estou encontrando orientação na internet para recuperar a senha do "sa" nesse contexto específico (SQL Server no Docker). Sabe como eu poderia fazer para resolver isso ?

**Comment by andrebaltieri** [comment, marked as answer]
Fica na parte `Env` da imagem que subiu:
![image](https://github.com/balta-io/forum/assets/965305/74525161-0963-42d1-a7d9-8c2221addde2)

Segue nosso tutorial para manter o padrão dos cursos:
https://balta.io/blog/sql-server-docker

**Reply by DrenokBR** [reply]
Pooo... brigadão Baltaaa !!! Achei aqui !!! Gratidão.

**Reply by DrenokBR** [reply]
Desculpa por ter te incomodado com algo simples.
Eu poderia ter pesquisado mais...
É que estou numa corrida contra o tempo aqui...

Muito obrigado !

**Reply by andrebaltieri** [reply]
@DrenokBR tranquilo!!!!
